index.bs

```diff
@@ -1813,6 +1821,17 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
                     </dl>
 
+            1. Let |attestationFormats| be a list of strings, initialized to the value of <code>|options|.{{PublicKeyCredentialCreationOptions/attestationFormats}}</code>.
+
+            1. If <code>|options|.{{PublicKeyCredentialCreationOptions/attestation}}</code>
+
+                    <dl class="switch">
+
+                        :   is set to {{AttestationConveyancePreference/none}}
+                        ::  Set |attestationFormats| be the single-element list containing the string &ldquo;none&rdquo;
+
+                    </dl>
+
```

Comment:
These two steps are missing from [§5.1.4. Use an Existing Credential to Make an Assertion](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1663.html#sctn-getAssertion) below, so the `|attestationFormats|` variable is uninitialized there.